inference-engine/ie_bridges/java/org/intel/openvino/Blob.java

```diff
@@ -0,0 +1,52 @@
+package org.intel.openvino;
+
+public class Blob extends IEWrapper {
+
+    protected Blob(long addr) {
+        super(addr);
+    }
+
+    public Blob(TensorDesc tensorDesc) {
+        super(GetBlob(tensorDesc.getNativeObjAddr()));
+    }
```

Comment:
since `CompoundBlob` uses `Blob` as a base class, it will have the same ctos like create blob using `tensorDesc`, right?
is it expected behavior?

Reply:
Where I can find the list of ```Blob``` methods which shouldn't be available for ```CompoundBlob```?